agent/scripts/compute-activity-scores.js

```
@@ -0,0 +1,154 @@
+// Computes community activity scores for all GSoC orgs
+// Run via GitHub Actions daily
+
+const fs = require('fs');
+const path = require('path');
+
+const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
```

ESLint

[error] 'process' is not defined.

(no-undef)